cmd/osd/main.go

```diff
@@ -477,16 +479,20 @@ func start(c *cli.Context) error {
 			return fmt.Errorf("Unable to Initialise Storage Policy Manager Instances %v", err)
 		}
 
+		poolServer := mockapi.NewMockOpenStoragePoolServer(gomock.NewController(
```

Comment:
This should not be mock but an actual server.

Reply:
removed this from osd main. nil server handler is now being handled.